amundsen_application/static/js/components/TableDetail/WatermarkLabel/index.tsx

```diff
@@ -10,27 +10,44 @@ interface WatermarkLabelProps {
 }
 
 class WatermarkLabel extends React.Component<WatermarkLabelProps> {
-
   constructor(props) {
     super(props);
-    this.getWatermarksLabel = this.getWatermarksLabel.bind(this);
   }
 
   render() {
     return (
-        <div className="watermark-label">{this.getWatermarksLabel(this.props.watermarks)}</div>
-      )
+      <div className="watermark-label">
+        <img className="range-icon" src="/static/images/watermark-range.png"/>
+        { this.getWatermarkText() }
+      </div>
+    );
   }
 
-  getWatermarksLabel(watermarks: Watermark[]) {
-    const low = watermarks.find((wtm) => wtm.watermark_type === "low_watermark");
-    const high = watermarks.find((wtm) => wtm.watermark_type === "high_watermark");
+  getWatermarkText = () => {
+    const low = this.props.watermarks.find((wtm) => wtm.watermark_type === "low_watermark");
+    const high = this.props.watermarks.find((wtm) => wtm.watermark_type === "high_watermark");
     if (low === undefined && high === undefined) {
-      return "Non Partitioned Table. Data available for all dates."
+      return (
+        <>
+          Non-Partitioned Table <br/>
+          Data available for all dates
+        </>
+      );
     }
-    return [low, high].map((wtm) => {
-      return moment(wtm.partition_value, "YYYY-MM-DD").format("MMM DD, YYYY");
-    }).join(" – ");
+
+    return (
+      <>
+        <div className="range-labels body-2">
+          From: <br/>
+          To:
+        </div>
+        <div className="range-dates body-2">
+          { low && moment(low.partition_value, "YYYY-MM-DD").format("MMM DD, YYYY") }
```

Comment:
I would create a `formatPartitionValue()` method so that for testing we can verify that the content here is equal to the output of that method. I think the issue of how to properly mock `moment-timezone` still exists so separating it allows us to at least mock `formatPartitionValue` and test everything else.